Middlewares/ST/STM32_USB_Device_Library/Class/MSC/Src/usbd_msc_data.c

```diff
@@ -91,62 +90,33 @@ uint8_t MSC_Page80_Inquiry_Data[LENGTH_INQUIRY_PAGE80] =
 /* USB Mass storage sense 6 Data */
 uint8_t MSC_Mode_Sense6_data[MODE_SENSE6_LEN] =
 {
-  0x22,
-  0x00,
-  0x00,
-  0x00,
-  0x08,
-  0x12,
-  0x00,
-  0x00,
-  0x00,
-  0x00,
-  0x00,
-  0x00,
-  0x00,
-  0x00,
-  0x00,
-  0x00,
-  0x00,
-  0x00,
-  0x00,
-  0x00,
-  0x00,
-  0x00,
-  0x00
+  0x03,     /* MODE DATA LENGTH. The number of bytes that follow. */
+  0x00,     /* MEDIUM TYPE. 00h for SBC devices. */
+  0x00,     /* DEVICE-SPECIFIC PARAMETER. For SBC devices: 
+             *   bit 7: WP. Set to 1 if the media is write-protected.
+             *   bits 6..4: reserved
+             *   bit 4: DPOFUA. Set to 1 if the device supports the DPO and FUA bits (used in caching)
+             *   bits 3..0: reserved*/
+  0x00      /* Put Product Serial number */
 };
 
 
 /* USB Mass storage sense 10  Data */
 uint8_t MSC_Mode_Sense10_data[MODE_SENSE10_LEN] =
 {
-  0x00,
-  0x26,
-  0x00,
-  0x00,
-  0x00,
-  0x00,
-  0x00,
-  0x00,
-  0x08,
-  0x12,
-  0x00,
-  0x00,
-  0x00,
-  0x00,
-  0x00,
-  0x00,
-  0x00,
-  0x00,
-  0x00,
-  0x00,
-  0x00,
-  0x00,
-  0x00,
-  0x00,
-  0x00,
-  0x00,
-  0x00
+  0x06,     /* MODE DATA LENGTH. The number of bytes that follow. */
+  0x00,     /* MEDIUM TYPE. 00h for SBC devices. */
+  0x00,     /* DEVICE-SPECIFIC PARAMETER. For SBC devices: 
+             *   bit 7: WP. Set to 1 if the media is write-protected.
+             *   bits 6..4: reserved
+             *   bit 4: DPOFUA. Set to 1 if the device supports the DPO and FUA bits (used in caching)
+             *   bits 3..0: reserved*/
+  0x00,     /* Reserved */
```

Comment:
Please update the comment. It should be `/* LONGLBA Set to zero */`.

Reply:
Seems you missed this one.

Reply:
The comment at line `115` has not been updated as requested. This would be probably fixed once version **v2.11.2** of the library integrated.